job-server/test/spark.jobserver/io/JobSqlDAOSpec.scala

```diff
@@ -228,36 +228,36 @@ class JobSqlDAOSpec extends TestJarFinder with FunSpecLike with Matchers with Be
 
       // First Test
       jobs.size should equal (2)
-      jobs.head should equal (expectedJobInfo)
+      jobs.last should equal (expectedJobInfo)
 
       // Second Test
       // Cannot compare JobInfos directly if error is a Some(Throwable) because
       // Throwable uses referential equality
       dao.saveJobInfo(jobInfoNoEndSomeErr)
       val jobs2 = dao.getJobInfos(2)
       jobs2.size should equal (2)
-      jobs2.head.endTime should equal (None)
-      jobs2.head.error.isDefined should equal (true)
-      intercept[Throwable] { jobs2.head.error.map(throw _) }
-      jobs2.head.error.get.getMessage should equal (throwable.getMessage)
+      jobs2.last.endTime should equal (None)
```

Comment:
Hmmm.... it seems the output of this DAO has gotten flipped....  not sure how that happened